rasa/shared/utils/common.py

```diff
@@ -5,32 +5,60 @@
 import logging
 from typing import Text, Dict, Optional, Any, List, Callable, Collection
 
+import rasa.shared.utils.io
+from rasa.shared.constants import NEXT_MAJOR_VERSION_FOR_DEPRECATIONS
+
+
 logger = logging.getLogger(__name__)
 
 
 def class_from_module_path(
-    module_path: Text, lookup_path: Optional[Text] = None
+    module_path: Text, lookup_path: Optional[Text] = None, ensure_class: bool = True
```

Comment:
As currently `ensure_class` only raises a warning, is there any situation we have it as `False`?

Reply:
no I don't think so. It's raising a deprecation warning, that will be transformed into an exception in the next major.

Reply:
🤔 I can't remember why exactly I introduced it (extra caution?). I think we can remove it actually: b6cceb5e90a